### ISSUE TITLE
Error running the CLI

### ISSUE THREAD

**Issue by nleroy917** [pepkit/looper#494]
I was trying to setup looper to dispatch a job for each file that I need to tokenize... this will really speed things up if I can get it to work properly.

However, I try to run the CLI in any capacity, and I get a an error: 
```
ImportError: cannot import name 'get_file_for_project' from 'pipestat.reports'
```

I just installed from pip (`pip install looper`) and it looks like I have version `1.7.0` installed.

**Comment by nleroy917** [comment]
@donaldcampbelljr helped me a bit here...

Downgrading to `pipestat==0.8.0` resolved the issue

**Comment by donaldcampbelljr** [comment]
This is now fixed with version 1.7.1
Thanks for flagging this.
